### ISSUE TITLE
Feature Request: Deselect All in interactive mode

### ISSUE THREAD

**Issue by limonte** [raineorshine/npm-check-updates#1202]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14`

---

It'd be great to be able to quickly deselect all upgradable deps in the interactive mode (`ncu -i`). In many cases I want to upgrade only a small subset of dependencies and ignore everything else. 

The whole list might be quite long, so ATM I have to press arrow down + space quite many times to deselect all manually.

Thanks for the amazing project 🙏

**Comment by raineorshine** [comment]
You can press `a` in interactive mode to select or deselect all.

This was conspicuously missing from the instructions. I updated the instructions and published to `v16.3.10` so this is clearer now.

**Comment by limonte** [comment]
Thank you so much, I missed this shortcut. Now people like me would be able to find this answer here: https://github.com/raineorshine/npm-check-updates/search?q=Deselect+All&type=issues

IMO this shortcut is quite important to be stated here:

<img width="781" alt="CleanShot 2022-10-09 at 17 43 16@2x" src="https://user-images.githubusercontent.com/6059356/194763161-8981f6f7-ed7b-4f6c-85ea-b6bbb965a14b.png">

```diff
- Choose which packages to update › Space to deselect. Enter to upgrade.
+ Choose which packages to update › Space to deselect. Enter to upgrade. a to toggle all.
```

**Comment by raineorshine** [comment]
Ah, yes. Those are the short instructions. There are longer instructions available with `--format group`. Maybe you are right though, as it is a time saver.

**Comment by raineorshine** [comment]
I decided to unify the interactive instructions so they are the same regardless of `--format`.

Published in `v16.3.11`.

```sh
% ncu -i
Upgrading /Users/raine/projects/ncu-issues/template/package.json
[====================] 2/2 100%

? Choose which packages to update ›
  ↑/↓: Select a package
  Space: Toggle selection
  a: Toggle all
  Enter: Upgrade

❯ ◉ ncu-test-tag   1.0.0  →   1.1.0
  ◉ ncu-test-v2   ^1.0.0  →  ^2.0.0
```

**Comment by limonte** [comment]
Amazing! You are a fantastic maintainer! 💪 💪 💪
